ios/A0Auth0.m

```diff
@@ -29,35 +30,49 @@ - (dispatch_queue_t)methodQueue
 }
 
 RCT_EXPORT_METHOD(showUrl:(NSString *)urlString callback:(RCTResponseSenderBlock)callback) {
-    NSURL *url = [NSURL URLWithString:urlString];
-    UIWindow *window = [[UIApplication sharedApplication] keyWindow];
-    SFSafariViewController *controller = [[SFSafariViewController alloc] initWithURL:url];
-    controller.delegate = self;
-    [self terminateWithError:RCTMakeError(@"Only one Safari can be visible", nil, nil) dismissing:YES animated:NO];
-    [window.rootViewController presentViewController:controller animated:YES completion:nil];
-    self.last = controller;
+    [self presentSafariWithURL:[NSURL URLWithString:urlString]];
     self.sessionCallback = callback;
 }
 
+RCT_EXPORT_METHOD(clearSession:(NSString *)domain federated:(BOOL)federated callback:(RCTResponseSenderBlock)callback) {
+    NSString *urlString = [NSString stringWithFormat:@"https://%@/v2/logout", domain];
```

Comment:
And TBH I'd rather build the full url in JS  and make this method a more generic one like show and close on load thing